cirq/google/calibration/phased_fsim.py

```diff
@@ -269,3 +329,27 @@ def parse_result(self, result: CalibrationResult) -> PhasedFSimCalibrationResult
         return FloquetPhasedFSimCalibrationResult(
             parameters=parsed, gate=self.gate, options=self.options
         )
+
+    @classmethod
+    def _from_json_dict_(
+        cls,
+        gate: Gate,
+        pairs: List[Dict[str, Qid]],
+        options: FloquetPhasedFSimCalibrationOptions,
+        **kwargs,
+    ) -> 'PhasedFSimCalibrationRequest':
+        """Magic method for the JSON serialization protocol.
+
+        Converts serialized dictionary into a dict suitable for
+        class instantiation."""
+        instantiation_pairs = tuple((entry['qubit_a'], entry['qubit_b']) for entry in pairs)
+        return cls(instantiation_pairs, gate, options)
+
+    def _json_dict_(self) -> Dict[str, Any]:
+        """Magic method for the JSON serialization protocol."""
+        return {
+            'cirq_type': 'FloquetPhasedFSimCalibrationRequest',
+            'pairs': [{'qubit_a': pair[0], 'qubit_b': pair[1]} for pair in self.pairs],
```

Comment:
Similarly here, if you use tuple instead of dict, this would be simpler. Up to you.

Reply:
Changed.